packages/nvidia_nat_strands/src/nat/plugins/strands/strands_callback_handler.py

```
@@ -0,0 +1,489 @@
+        try:
+            params = getattr(model_instance, "params", {})
+            if isinstance(params, dict):
+                model_params = {
+                    "temperature": params.get("temperature"),
+                    "max_tokens": params.get("max_tokens"),
+                    "top_p": params.get("top_p"),
+                    "stream": params.get("stream", True),
+                }
+                # Remove None values
+                model_params = {k: v for k, v in model_params.items() if v is not None}
+        except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)